subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -48,6 +48,9 @@ config BT_LLL_VENDOR_NORDIC
 	select BT_CTLR_CHAN_SEL_2_SUPPORT
 	select BT_CTLR_MIN_USED_CHAN_SUPPORT
 	select BT_CTLR_SCA_UPDATE_SUPPORT
+	select BT_CTLR_SYNC_TRANSFER_RECEIVER_SUPPORT
+	select BT_CTLR_SYNC_TRANSFER_SENDER_SUPPORT
+	select BT_TICKER_LAZY_GET
```

Comment:
```suggestion
```

Reply:
There is line below like this `select BT_TICKER_LAZY_GET if BT_CTLR_ADV_PERIODIC || BT_CTLR_CENTRAL_ISO` ; append `|| BT_CTLR_SYNC_TRANSFER_SENDER`  to it